### PR TITLE
Added Laravel 6 support

### PR DESCRIPTION
**Pull request by realrashid**
Hey Man!

I just added Laravel 6 support on this great package and also i have make some minor changes on Readme.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": "^7.2",
+        "laravel/framework": "~5.4.0|~5.8.0|~5.9.0|^6.0"
```
Comment (gizburdt) [inline composer.json]: Please remove the php requirement, and change laravel/framework to ^5.4|^6.0.

Thanks :)

Comment (realrashid) [reply composer.json]: Ok sure.

Comment (realrashid) [reply composer.json]: I am setting the php as default. 
"php": ">=5.6"

File: composer.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": ">=5.6",
+        "laravel/framework": "^5.4|^6.0"
```
Comment (realrashid) [inline composer.json]: I changed the php requirement as default.

**Review by realrashid — COMMENTED** [review]
Updated Please check.

File: composer.json (hunk 2 above)
Comment (roelofr) [inline composer.json]: To make it easier to extend (to stuff like Laravel Lumen), I'd suggest using the [illuminate repository](https://github.com/illuminate). This makes the package more portable.

```suggestion
        "illuminate/support": "^5.4|^6.0",
        "illuminate/database": "^5.4|^6.0"
```

**Pull request merged**